src/Stryker.Core/Stryker.Core.UnitTest/Mutators/AbstractMutatorTests.cs

```diff
@@ -84,7 +84,11 @@ public void ShouldNotMutateIfMutationLevelIsLow()
             var target = new ExampleMutator(MutationLevel.Complete);
```

Comment:
ExampleMutation?

Reply:
Well no this is definitely a mutator. It implements the `IMutator` interface